### DISCUSSION TITLE
Cannot find addresses that can be found in the Nominatim

### DISCUSSION THREAD

**Discussion by AdrianVovk** [komoot/photon#795] [Q&A]
Original issue is here: https://gitlab.gnome.org/GNOME/gnome-maps/-/issues/632

Basically, there are certain addresses that I can find on OpenStreetMap, but not in GNOME Maps (which uses Photon). I opened an issue for GNOME Maps and was redirected here instead.

For example: look up `4400 Zachary Court, Dublin OH 43017`. The [OpenStreetMap Nominatim](https://nominatim.openstreetmap.org/ui/search.html?q=4400+Zachary+Court%2C+Dublin+OH+43017) finds the house without issue. [Photon](https://photon.komoot.io/), on the other hand, just finds the street.

For this example address I found it's not the end of the world; the street is tiny. But if I look up my home, I instead end up looking at the midpoint of my street, > 1 km away

**Comment by lonvia** [comment]
This housenumber is not in OpenStreetMap. Nominatim uses additional housenumber interpolations from Tiger. These are not imported into Photon.

**Reply by AdrianVovk** [reply]
> This housenumber is not in OpenStreetMap

Yes, this is true. But as far as I can tell most of the house numbers in my area are not on the map. Unfortunately it makes the map pretty much useless, if most addresses are not find-able...

> These are not imported into Photon

Why not? Could they be?

**Reply by mtmail** [reply]
> Could they be [imported into Photon]?

TIGER (US Census) data is interpolated ranges of house numbers, not individual addresses. With "500-750" it's not possible to know which house numbers really exist. Photon software can't deal with interpolated data.

> most of the house numbers in my area are not on the map. Unfortunately it makes the map pretty much useless,

Any help adding house numbers in your area to OpenStreetMap is much appreciated.
